de.json

```
@@ -895,7 +895,7 @@
         "Inspector.Mesh.UV_Multiplicator" : "Multiplikator:",
         "Inspector.Mesh.ScaleUVs" : "Skaliere UV's",
 
-        "Inspector.MeshRenderer.MergeByMaterial" : "Submeshes mit demselben Material zusammenführen",
+        "Inspector.MeshRenderer.MergeByMaterial" : "Submeshes mit dem selben Material zusammenführen",
```

The old commit was grammatically correct. 
```suggestion
        "Inspector.MeshRenderer.MergeByMaterial" : "Submeshes mit demselben Material zusammenführen",
```